p/themes/base-theme/template.css

```diff
@@ -373,16 +369,91 @@ a.btn {
 	cursor: grab;
 }
 
+/*=== Scrollbar */
+.ss-wrapper {
+	overflow: hidden;
+	width: 100%;
+	height: 100%;
+	position: relative;
+	z-index: 1;
+	float: left;
+}
+
+.ss-content {
+	height: 100%;
+	width: calc(100% + 18px);
+	padding: 0 0 0 0;
+	position: relative;
+	overflow-y: scroll;
+	box-sizing: border-box;
+}
+
+.ss-content.rtl {
+	width: calc(100% + 18px);
+	right: auto;
+}
+
+.ss-scroll {
+	position: relative;
+	background: rgba(0, 0, 0, 0.1);
+	width: 9px;
+	border-radius: 4px;
+	top: 0;
+	z-index: 2;
+	cursor: pointer;
+	opacity: 0;
+	transition: opacity 0.25s linear;
+}
+
+.ss-hidden {
+	display: none;
+}
+
+.ss-container:hover .ss-scroll,
+.ss-container:active .ss-scroll {
+	opacity: 1;
+}
+
+.ss-grabbed {
+	-o-user-select: none;
+	-ms-user-select: none;
+	-moz-user-select: none;
+	-webkit-user-select: none;
+	user-select: none;
+}
+
+#sidebar {
+	scrollbar-width: thin;
+	scrollbar-color: rgba(0, 0, 0, 0.0) rgba(0, 0, 0, 0.0);
+}
+
+#sidebar:hover {
+	scrollbar-width: thin;
+	scrollbar-color: rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.0);
+}
+
+#sidebar::-webkit-scrollbar {
+	width: 8px;
+}
+#sidebar::-webkit-scrollbar-thumb {
+	border-radius: 5px;
+	display: none;
+}
+#sidebar:hover::-webkit-scrollbar-thumb {
+	background: rgba(0, 0, 0, 0.3);
+	display: unset;
+}
+
 /*=== Tree */
 .tree {
 	margin: 0;
-	padding: 0 0 2em 0;
 	list-style: none;
 	text-align: left;
-}
-
-.treepadding {
-	padding: 0 0 15em 0;
+	position: absolute;
+	overflow-y: auto;
+	overflow-x: hidden;
+	width: 300px;
+	height: 90vh;
```

Comment:
The Firefox bug seems to be due to this property

Reply:
Looking at the commit history, the height property here was when I was thinking that I could break the scrolling out without javascript. It's removal certainly doesn't seem to break anything.